### PR TITLE
+doc New UI for the autocomplete

### PR DESCRIPTION
**Pull request by ElPicador**
A new UI for the autocomplete, a bit nicer I think :)

### CODE REVIEW

**Comment by typesafehub-validator** [comment]
Hi @ElPicador,

Thank you for your contribution! We really value the time you've taken to put this together.

Before we proceed with reviewing this pull request, please sign the Typesafe Contributors License Agreement:

[http://www.lightbend.com/contribute/cla](http://www.lightbend.com/contribute/cla)

File: akka-docs/_sphinx/themes/akka/layout.html [hunk 1]
```diff
@@ -219,10 +218,11 @@
     indexName: 'akka_io',
     inputSelector: '#search',
     algoliaOptions: {
-      hitsPerPage: 10,
-      facetFilters: '[' +
-      '["language:' + lang + '","language:general"]' +
-      ',"version:2.4.2"]'
+      hitsPerPage: 5,
+      facetFilters: '[' + '["language:' + lang + '","language:general"]' + ',"version:2.4.2"]',
```
Comment (2beaucoup) [inline akka-docs/_sphinx/themes/akka/layout.html]: 2.4.4?

**Comment by typesafehub-validator** [comment]
Hi @ElPicador,

Thank you for your contribution! We really value the time you've taken to put this together.

Before we proceed with reviewing this pull request, please sign the Typesafe Contributors License Agreement:

[http://www.lightbend.com/contribute/cla](http://www.lightbend.com/contribute/cla)

File: akka-docs/_sphinx/themes/akka/layout.html (hunk 1 above)
Comment (ElPicador) [inline akka-docs/_sphinx/themes/akka/layout.html]: I kept the same version as before.

Comment (ktoso) [inline akka-docs/_sphinx/themes/akka/layout.html]: Oh, that's a bug I did I think! Glad that we noticed it here, I'll fix on prod and after merging this

**Comment by ElPicador** [comment]
I accepted and signed the CLA. Is there something more to do to pass the check?

**Comment by ktoso** [comment]
That's all we needed here, thank you!

**Comment by ktoso** [comment]
And I'll fix the 2.4.2 being hardcoded bug right away

File: akka-docs/_sphinx/themes/akka/layout.html (hunk 1 above)
Comment (ktoso) [inline akka-docs/_sphinx/themes/akka/layout.html]: This is now fixed btw

**Pull request merged**